### DISCUSSION TITLE
Is it possible to make the content taken from an HTML document?

### DISCUSSION THREAD

**Discussion by GreenEyeInvestor** [ReactTooltip/react-tooltip#1116] [Q&A]
I'm using next.js and I need to output the content in HTML itself, and not generate a hint at the user's request.
I have a button code and a block with content on my page, is it possible to link the content to the tooltip somehow? Roughly speaking, I just need to show and hide the block, naturally with positioning.

Here's the button
`<button data-tooltip-id="search">Button</button>`

And at the moment the content itself

```
<Tooltip
    id="tooltip"
    positionStrategy={"fixed"}
    closeOnScroll={true}
    closeOnEsc={true}
    place={"bottom"}
    className={}
    opacity={.94}
    openOnClick={true}
    closeOnResize={true}
>content</Tooltip>
```
The problem is that the content is formed only when you hover over the button, and is not in the structure of the document

**Comment by gabrieljablonski** [comment]
@GreenEyeInvestor sorry for taking so long to respond. When you sent this my GitHub notifications were not working properly so I totally missed this post, since it also is kinda hidden in the discussions tab.

From your description, I'm not 100% sure what you're trying to accomplish, but my best guess is that you're trying to use the tooltip without the usual hover/unhover interaction from the user? If so, check out [v5.23.0](https://github.com/ReactTooltip/react-tooltip/releases/tag/v5.23.0) (and [the docs for the imperative mode](https://react-tooltip.com/docs/examples/imperative-mode)). That might be what you're trying to do.

If that's not it, please try and give some more details.
